qiskit/primitives/base_estimator.py

```diff
@@ -145,6 +147,7 @@ def __init__(
                 will be bound. Defaults to ``[circ.parameters for circ in circuits]``
                 The indexing is such that ``parameters[i, j]`` is the j-th formal parameter of
                 ``circuits[i]``.
+            run_options: runtime options.
```

Comment:
Could we add a bit more information here, like "The runtime options for individual estimator executions, such as the number of shots." ? 🙂

Reply:
Thanks. Whether number of shots is supported or not is implementation-dependent, so it is not known at the base class level and cannot be written here. 

> The runtime options for individual estimator executions.

is fine.